api/v1beta1/flinkcluster_types.go

```diff
@@ -568,6 +574,22 @@ type FlinkClusterStatus struct {
 	// The status of savepoint progress
 	Savepoint *SavepointStatus `json:"savepoint,omitempty"`
 
+	// When the controller creates new ControllerRevision, it generates hash string from the FlinkCluster spec
+	// which is to be stored in ControllerRevision and uses it to compose the ControllerRevision name.
+	// Then the controller updates nextRevision to the ControllerRevision name.
```

Comment:
I don't see where the controller updates status.nextRevision, could you point me to it?

Reply:
https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/blob/2ca1acecdf486c054b14993d48c598d124b95c5a/controllers/flinkcluster_updater.go#L930-L935